app/src/main/res/layout/activity_main_kotlin.xml

```diff
@@ -6,4 +6,16 @@
     android:layout_height="match_parent"
     tools:context=".MainActivityKotlin">
 
+    <FrameLayout
+        android:id="@+id/fragment_container"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <fragment
+            android:id="@+id/fragment"
+            android:name="com.example.weatherapp.FragmentOne"
```

Comment:
You can load the fragment statically for now, but when we will have more fragments we will have to load them dynamically with fragment manager

Reply:
I read the same thing before I uploaded the changes. I'll do a bit of research on fragment managers so hopefully, we can update it in the next commit